typescript/infra/config/registry.ts

```diff
@@ -64,13 +70,43 @@ export function getRegistry(): FileSystemRegistry {
   return registry;
 }
 
-export function getGithubRegistry(): GithubRegistry {
-  return new GithubRegistry({
-    proxyUrl: REGISTRY_PROXY,
-    logger: rootLogger.child({ module: 'infra-registry' }),
+/**
+ * Creates a new MergedRegistry using the provided URIs
+ * The intention of the MergedRegistry is to join the common data
+ * from a primary URI (such as a filesystem)
+ * and an override one (such as a github directory)
+ * @returns a new MergedRegistry
+ */
+export function getMergedRegistry(
```

Comment:
Could we just import the CLI function here and then call this with `primaryRegistryUri` from `DEFAULT_REGISTRY_URI`? Why introduce another ENV var and this logic?

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/5398